solver/src/naive_solver.rs

```diff
@@ -19,17 +20,11 @@ impl Solver for NaiveSolver {
         _gas_price: f64,
     ) -> Result<Option<Settlement>> {
         let mut limit_orders = Vec::new();
-        let mut uniswaps = HashMap::new();
+        let uniswaps = extract_deepest_amm_liquidity(&liquidity);
```

Comment:
Nit: Maybe swap these two lines so that limit_orders are declared and populated together without uniswaps in between